lms/djangoapps/teams/static/teams/js/views/my_teams.js

```diff
@@ -0,0 +1,30 @@
+;(function (define) {
+    'use strict';
+
+    define(['backbone', 'gettext', 'teams/js/views/teams'],
+        function (Backbone, gettext, TeamsView) {
+            var MyTeamsView = TeamsView.extend({
+                render: function() {
+                    TeamsView.prototype.render.call(this);
+                    if (this.collection.length === 0) {
+                        this.$el.append('<p>' + gettext('You are not currently a member of any teams.') + '</p>');
```

Comment:
@andy-armstrong I would just make "teams" singular: "You are not currently a member of any team".